src/components/ResumeUploadBtn.js

```diff
@@ -0,0 +1,38 @@
+import React, { useRef } from 'react'
+import { Button } from '../components/Input'
+import { QuestionHeading } from '../components/Typography'
+import styled from 'styled-components'
+import { TextInputLikeErrorMsg } from './Common'
+
+const ResumeContainer = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+export default ({ onChange, isValid, hint, errorMsg }) => {
+  const inputFile = useRef()
+
+  const handleClick = () => {
+    inputFile.current.click()
+  }
+
+  const resumeFile = <input ref={inputFile} type="file" hidden onChange={onChange} />
+  const button = (
+    <Button color="tertiary" onClick={handleClick}>
+      Upload
+    </Button>
+  )
```

Comment:
It's not best practice to define components inside other components like this. I'm not 100% sure if the concept applies to functional components but it's better to get in the habit of not doing it.

I believe it has something to do with recreating new child components every time the component re-renders

Just move these components above line 12, and pass in the things u need like inputFile, onChange etc as props

Reply:
oooh makes sense thank you. i think i'll wait for Jenny to comment on any other suggestions first and I'll just make all those changes together later for the next commit

Reply:
ok i'm kinda confused LMAO uhh i'm not super clear on what you mean by "pass in the things u need like inputFile, onChange etc as props" and how to do that hehe. so if i wanna move resumeFile and button outside of the component to the top of the class, how do I access the onChange prop as well as the handleClick, inputFile which are defined inside the component?

Reply:
```
const ResumeFile = ({inputFile, onChange}) => {...}

// inside default export
<ResumeFile inputFile={inputFile} onChange={onChange} />
```

Reply:
omg whatttt ok that's cool didn't think about that big thanks sir